diff --git a/node_modules/codecake/codecake.js b/node_modules/codecake/codecake.js
index 18e1dbd..144c0ed 100644
--- a/node_modules/codecake/codecake.js
+++ b/node_modules/codecake/codecake.js
@@ -78,6 +78,19 @@ export const create = (parent, options = {}) => {
     // 'plaintext-only' mode is not supported in Firefox
     if (!options?.readOnly && editor.contentEditable !== "plaintext-only") {
         editor.setAttribute("contenteditable", "true");
+        editor.addEventListener("paste", event => {
+            let insertText = event.clipboardData.getData("text/plain");
+            event.preventDefault();
+
+            // Insert text at cursor position
+            const sel = window.getSelection();
+            const range = sel.getRangeAt(0);
+            range.deleteContents();
+            range.insertNode(document.createTextNode(insertText));
+
+            // Update editor
+            update(10);
+        })
     }
     if (options?.lineWrap) {
         editor.classList.add("codecake-linewrapping");
