diff --git a/node_modules/esbuild-svelte/dist/index.js b/node_modules/esbuild-svelte/dist/index.js
index 869b275..f0d6c2c 100644
--- a/node_modules/esbuild-svelte/dist/index.js
+++ b/node_modules/esbuild-svelte/dist/index.js
@@ -180,6 +180,11 @@ function sveltePlugin(options) {
           let { js, css, warnings } = await (async () => {
             if (SVELTE_VERSION === 5 && SVELTE_MODULE_FILTER.test(filename)) {
               const { compileModule } = await import("svelte/compiler");
+              source = source.replace("as $", "as temp");
+              source = source.replace("$.autoDownloadMissingLibs", "temp.autoDownloadMissingLibs");
+              source = source.replace("$=new Blob", "temp=new Blob");
+              source = source.replace(".match($)", ".match(temp)");
+              source = source.replace("createObjectURL($)", "createObjectURL(temp)");
               return compileModule(source, {
                 ...moduleCompilerOptions,
                 filename
